changes/a91b110f56bd02fb9b95bffba5727c2c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+desc: Added support to the Oauth2 providers to use ``client_assertion`` values to
+  authenticate to Oauth2 endpoints. These can be used instead of providing a
```

Comment:
```suggestion
desc: Added support to the OAuth2 providers to use ``client_assertion`` values to
  authenticate to OAuth2 endpoints. These can be used instead of providing a
```